client/packages/android/README.MD

```diff
@@ -88,18 +88,41 @@ For this reason `ExtendedWebViewClient` was created to manually inject the requi
 
 ## Extra
 
-The cert plugin (`app/src/main/java/org/openmsupply/client/certplugin/CertPlugin.java`) allows the web client to make https request to the remote-server using a self signed certificate. It needs to extend `ExtendedWebViewClient` rathern then `WebViewClient` to allow for Capacitor Modification 2. to work.
+The cert plugin (`app/src/main/java/org/openmsupply/client/certplugin/CertPlugin.java`) allows the web client to make https request to the remote-server using a self signed certificate. It needs to extend `ExtendedWebViewClient` rather than `WebViewClient` to allow for Capacitor Modification 2. to work.
 
 `RemoteServer.java` is jni mapping to functions defined in `server/android/src/android.rs`, it is started and stopped in `MainActivity.java`
 
 ## CAP
 
-Capacitor comes with cli, we mainly use `npx cap copy` (or `yarn apply-config`), this moves bundled assets to app/src/main/assets/public. Usually capacitor would bundle web app within the APK, since we are serving front end bundle with server we don't need to move them (you will not that in capacitor.config.ts `webDir` is pointed to a notexistet directory). Capacitor also moves pre requisites to cordova plugins to that folder, which is automatically injected into served `html`. Another taks of `npx cap copy` is to copy configuration files `capacition.config.ts` is translated to JSON file and moved to app/src/main/assets
+Capacitor comes with cli, we mainly use `npx cap copy` (or `yarn apply-config`), this moves bundled assets to app/src/main/assets/public. Usually capacitor would bundle web app within the APK, since we are serving front end bundle with server we don't need to move them (you will not that in capacitor.config.ts `webDir` is pointed to a non-existant directory). Capacitor also moves pre requisites to cordova plugins to that folder, which is automatically injected into served `html`. Another task of `npx cap copy` is to copy configuration files `capacitor.config.ts` is translated to JSON file and moved to app/src/main/assets
 
-`app/src/main/assets/public` directory is typically not commited, but since it's only going to have cordova artifacts, it is in our case (to reduce setup). 
+`app/src/main/assets/public` directory is typically not committed, but since it's only going to have cordova artifacts, it is in our case (to reduce setup). 
 
 `npx cap copy` or `yarn apply-config` should only be run when cordova plugins are added or updated or when updating capacitor.config.ts
 
+### Plugins
+
+When adding plugins, the standard process (`npx cap sync` or `yarn apply-config`) is not working and I've had to manually add them:
```

Comment:
Thank you Mark, i had problems with `npx cap sync` also, with some dependencies being added (this is even before the refactor to discovery), I thought I fixed it in the changes but nope, adding plugins was still not working (found out not long ago but never fixed). Your instructions look very good